app/Model/Attribute.php

```diff
@@ -2515,6 +2498,10 @@ public function captureAttribute($attribute, $eventId, $user, $objectId = false,
                 unset($attribute['sharing_group_id']);
             }
         }
+        if (isset($attribute['breakOnDuplicate']) && $attribute['breakOnDuplicate'] === false) {
+            $this->data['Attribute'];
```

Comment:
Doesn't seems to do anything. Delete the line?

Reply:
good catch